db.js

```diff
@@ -258,6 +299,12 @@ function upgrade (db) {
   actors.createIndex(UPDATED_FIELD, UPDATED_FIELD)
   actors.createIndex(URL_FIELD, URL_FIELD)
 
+  if (!db.objectStoreNames.contains(FOLLOWED_ACTORS_STORE)) {
```

Comment:
This if statement shouldn't be necessary, try removing it and clearing your db in the devtools to see if it upgrades correctly